data/src/main/java/com/whyranoid/data/model/challenge/ChallengeDetailResponse.kt

```diff
@@ -1,29 +1,38 @@
 package com.whyranoid.data.model.challenge
 
+import com.whyranoid.domain.model.challenge.Challenge
 import com.whyranoid.domain.model.challenge.ChallengeType
+import com.whyranoid.domain.util.EMPTY
 
 data class ChallengeDetailResponse(
     val challenge: ChallengeFromServer,
     val walkies: List<Walkie>
 ) {
-    fun toChallenge(): com.whyranoid.domain.model.challenge.Challenge {
-        val participants = walkies.map { it.toUser() }
-        return com.whyranoid.domain.model.challenge.Challenge(
-            badge = challenge.badge.toBadge(),
-            calorie = challenge.calorie,
-            challengeType = ChallengeType.getChallengeTypeByString(challenge.category),
-            id = challenge.challengeId.toLong(),
-            contents = challenge.content,
-            imageUrl = challenge.img,
-            title = challenge.name,
-            process = challenge.progress,
-            participants = participants,
-            participantCount = participants.size,
-            period = challenge.period,
-            distance = challenge.distance,
-            endTime = challenge.endTime,
-            startTime = challenge.startTime,
-            status = challenge.status
+    fun toChallenge(): Challenge {
+        val challenge = Challenge(
+            challenge.accCalories,
+            challenge.accCount,
+            challenge.accDistance,
+            challenge.accTime,
+            challenge.badge.toBadge(),
+            challenge.calorie,
+            ChallengeType.getChallengeTypeByString(challenge.category ?: String.EMPTY),
```

Comment:
nit) 다음과 같이 줄여 볼 수 있을 것 같습니다!
```suggestion
            ChallengeType.getChallengeTypeByString(challenge.category.orEmpty()),
```

Reply:
!! orEmpty 좋네요! 코틀린에서 자체적으로 있는 메서드인가요?